the-blue-alliance-ios/View Elements/Events/RankingTableViewCell.swift

```diff
@@ -46,9 +65,18 @@ class RankingTableViewCell: UITableViewCell {
         }
     }
     
+    func setupWLTLabel(ranking: EventRanking) {
+        if let wins = ranking.wins, let losses = ranking.losses, let ties = ranking.ties {
+            WLTLabel?.text = "(\(wins)-\(losses)-\(ties))"
+        }
+        
+    }
+ 
     @IBOutlet public var rankLabel: UILabel?
     @IBOutlet private var numberLabel: UILabel?
     @IBOutlet private var nameLabel: UILabel?
-    @IBOutlet private var detailLabel: UILabel?
-    @IBOutlet private var detailLabelWidth: NSLayoutConstraint?
+    @IBOutlet private var WLTLabel: UILabel?
```

Comment:
True. It's something I do in my own code for acronyms because otherwise I tend to mentally pronounce them wrong, but I'm happy to change it here.